### DISCUSSION TITLE
Given a page's title, how to get the username of the page's creator ?

### DISCUSSION THREAD

**Discussion by hugolpz** [kanasimi/wikiapi#64] [General]
👆🏼

**Comment by kanasimi** [comment]
Well... I didn't write this function... This can be obtained by the oldest revision.

**Reply by hugolpz** [reply]
True ! Thanks for the hint. Also, I'm doing thousand (s?) edits on meta using WikiapiJS ! #HappyJuniorDev

**Reply by hugolpz** [reply]
I go with `await wiki.page('Universe', { revisions: 100 })` (many revisions) and I get the username from the earliest one, right ?

**Reply by kanasimi** [reply]
See https://www.mediawiki.org/w/api.php?action=help&modules=query%2Brevisions
``` JavaScript
page_data = await wiki.page('Universe', {rvdir: 'newer', rvprop: 'user'});
CeL.wiki.content_of.revision(page_data, 0);
```

**Reply by hugolpz** [reply]
This works !
```javascript
var page_data = await targetWiki.page(title, {/*prop: 'revisions',*/ rvdir: 'newer', rvprop: 'user', rvlimit: 1});
console.log('CREATOR: ', page_data.revisions[0].user)
```
